man/fi_mr.3.md

```diff
@@ -554,18 +558,22 @@ struct fi_mr_dmabuf {
 	int      fd;
 	uint64_t offset;
 	size_t   len;
+	void     *base_addr;
 };
 ```
 The fd is the file descriptor associated with the DMA-buf region.  The
 offset is the offset into the region where the memory registration should
 begin.  And len is the size of the region to register, starting at the
-offset.  The selection of dmabuf over the mr_iov field is controlled by
+offset. The base_addr is the base virtual address of the memory region to
+be registered. If a base virtual address is not available (because, for
+example, the calling process has not mapped the memory region into its
+address space), base_addr can be set to NULL. If both base_addr and
+offset are provided, then the page offset of both the base_addr and
+offset parameters must be identical.
```

Comment:
base address is supposed to be the page aligned starting address of the allocation.  offset is the difference between the virtual address and the base address.
```
virtual address = base address + offset
```

Reply:
I see, thanks